src/modules/__tests__/renderValue.spec.js

```diff
@@ -251,6 +254,57 @@ const tests = [
         round: true,
         dgs: DGS_NONE,
     },
+    // Phone number
+    {
+        value: '99777888',
+        expected: '99777888',
+        valueType: VALUE_TYPE_PHONE_NUMBER,
+        round: true,
+        dgs: DGS_SPACE,
+    },
+    {
+        value: '(303)9977888',
+        expected: '(303)9977888',
+        valueType: VALUE_TYPE_PHONE_NUMBER,
+        round: true,
+        dgs: DGS_NONE,
+    },
+    {
+        value: '303-8485-9922',
+        expected: '303-8485-9922',
+        valueType: VALUE_TYPE_PHONE_NUMBER,
+        round: true,
+        dgs: DGS_COMMA,
+    },
+    {
+        value: '303  848  9922',
+        expected: '303 848  9922',
```

Comment:
This happens because of the regex here: https://github.com/dhis2/analytics/blob/master/src/modules/renderValue.js#L62
I discussed this previously with Austin who wrote this, but nothing really came out of it, and since no bugs have been reported, I've just left it as is (I'm very hesitant to go changing regexes).

So this test was added specifically to illustrate the regex behavior. Shall I add a comment?